01-js/easy/anagram.js

```diff
@@ -9,6 +9,17 @@
 
 function isAnagram(str1, str2) {
 
+  str1 = str1.toLowerCase();
+  str2 = str2.toLowerCase();
+
+  str1 = str1.split("").sort().join();
+  str2 = str2.split("").sort().join();
+
+  if(str1==str2)
+    return true;
+  else  
+    return false;
```

Comment:
Clean way to write like this
```js
return str1 === str2
```

- Use === instead of == equals operator
- str1 === str2 itself returns boolean, which you can use to return

Reply:
Thanks @manassahoo-dev, I'll keep that in mind.